circle.yml

```diff
@@ -16,7 +16,7 @@ dependencies:
     - tar -xzf sc-latest-linux.tar.gz
     - sudo pip install --upgrade pip
     - sudo pip install gunicorn
-    - npm run install-openfisca
+    - pip install --user --upgrade -r openfisca/requirements.txt
```

Comment:
I would much rather have CI mimick the suggested installation procedure, and set up a virtualenv then call `npm run install-openfisca` rather than having two ways of installing.